ansible/roles/dev-desktop/defaults/main.yml

```diff
@@ -9,3 +9,7 @@ vars_user_quota_gb: 50
 # For Ubuntu hosts, change this in a host variable to:
 # "linux-modules-extra-virtual"
 vars_user_quota_linux_package: "linux-modules-extra-aws"
+
+# For Ubuntu hosts, change this in a host variable to:
+# "linux-tools-common"
+vars_perf_linux_package: "linux-tools-aws"
```

Comment:
Can we change both variables to `vars_linux_kernel_flavor` that can be "aws" or "virtual"? `linux-tools-virtual` exists as well, so it should be just possible to suffix the package names with the kernel flavor.